lang/api/build.gradle.kts

```diff
@@ -23,6 +25,17 @@ dependencies {
     stdLib(kotlin("stdlib"))
 }
 
+tasks.withType<KotlinJvmCompile>().named { it.contains("Test") }.configureEach {
```

Comment:
nit: I think it can be put as general as `yatagan.base-module`, because it doesn't hurt to compile tests with Java 11 compatibility for every module.